source/_components/aurora_abb_powerone.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+title: "Aurora ABB Powerone PV Inverter Sensor"
+description: "Instructions on how to integrate an Aurora ABB Powerone solar inverter within Home Assistant."
+date: 2019-06-27 23:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category:
+  - Sensor
+  - Energy
+ha_release: 0.96
+ha_iot_class: Local Polling
+---
+
+This implements a direct RS485 connection to a solar inverter in the 
+PVI-3.0/3.6/4.2-TL-OUTD ABB series, and may work on others.
+The inverter was formerly made by PowerOne who got taken over by ABB.
+
+The TCP/IP method of commuicating with inverters is supported by the 
+python library, but not by this implementation of the homeassistant component.
+
+The component provides a single sensor which reports the live power output
+in watts.
+
+Note the PV inverter will be unresponsive to communications when in darkness.
+
+This is caught by the implementation which returns 'None' if there is no 
```

Comment:
How the code/implementation works, is not relevant in this case. Most users don't know what `None` means.
Please change this or remove it.

Reply:
Updated in PR patch-1